controllers/endpoints.js

```diff
@@ -1,24 +1,39 @@
 const fleetService = require('../services/fleet');
+const formatDate = require('../public/js/endpoint.js'); 
```

Comment:
Not recommended. Frontend <--> Backend imports == 💥

Reply:
removed - no longer needed